tests/stubtest_whitelists/py3_common.txt

```diff
@@ -573,6 +570,13 @@ zipfile.ZipExtFile.read
 zipfile.ZipExtFile.readline
 zlib.compressobj
 
+# Modules that do not exist on Linux systems and cannot be checked by our CI process.
```

Comment:
nit: I'd move this below the "Whitelist entries that cannot or should not be fixed" header.

Additionally, if we're looking to try to minimise "unused whitelist entry" errors if someone attempts to run stubtest_test.py on Windows locally, we might want to modify the list as follows, using the method described in https://github.com/python/mypy/pull/8502#discussion_r389196956:
```
(_msi)?
(_winapi)?
(msilib(\.[a-z]+)?)?
(msvcrt)?
(winsound)?
```